diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index f62598da9b3a9fa735144a2cc0d8e5cb8d00b9d9..81f1b47ad75b2d69c7933b282464a238a476ca24 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -204,6 +204,7 @@ exports.networks = [
     { chainId: 7560n, shortName: 'cyeth' },
     { chainId: 7700n, shortName: 'canto' },
     { chainId: 7771n, shortName: 'tbitrock' },
+    { chainId: 7897n, shortName: 'arena-z' },
     { chainId: 8192n, shortName: 'tqf' },
     { chainId: 8194n, shortName: 'ttqf' },
     { chainId: 8217n, shortName: 'cypress' },
