diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 09d831da880738f9f92318a1b8944232f773caf7..156ab472da26fbad05130895bdc7c3c78167cf91 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -3,6 +3,9 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.networks = void 0;
 // https://github.com/ethereum-lists/chains/tree/master/_data/chains
 exports.networks = [
+    { chainId: 2522n, shortName: 'fraxtal-testnet' },
+    { chainId: 8008n, shortName: 'polynomial' },
+    { chainId: 80008n, shortName: 'polynomialSepolia' },
     { chainId: 1n, shortName: 'eth' },
     { chainId: 3n, shortName: 'rop' },
     { chainId: 4n, shortName: 'rin' },
