diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 4937a8edcfb3981d3974d2a0fa9bafdb92cde131..5b5f5fc76e7b36a6c0789a9b835a460a0e74d9e0 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -51,6 +51,14 @@ exports.networks = [
     { chainId: 133n, shortName: 'HSKT' },
     { chainId: 137n, shortName: 'matic' },
     { chainId: 146n, shortName: 'sonic' },
+    { chainId: 185n, shortName: 'mint' },
+    { chainId: 1946n, shortName: 'minato' },
+    { chainId: 2522n, shortName: 'fraxtal-testnet' },
+    { chainId: 763373n, shortName: 'ink-sepolia' },
+    { chainId: 1868n, shortName: 'soneium' },
+    { chainId: 1923n, shortName: 'swell-l2' },
+    { chainId: 1924n, shortName: 'swell-l2-testnet' },
+    { chainId: 5330n, shortName: 'superseed' },
     { chainId: 148n, shortName: 'shimmerevm' },
     { chainId: 155n, shortName: 'tenet-testnet' },
     { chainId: 169n, shortName: 'manta' },
