diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 4937a8edcfb3981d3974d2a0fa9bafdb92cde131..771c9f22571c135d26e93ed8eee27f4a959202ba 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -57,6 +57,13 @@ exports.networks = [
     { chainId: 177n, shortName: 'hsk' },
     { chainId: 179n, shortName: 'abey' },
     { chainId: 181n, shortName: 'water' },
+    { chainId: 185n, shortName: 'mint' },
+    { chainId: 1868n, shortName: 'soneium' },
+    { chainId: 1946n, shortName: 'minato' },
+    { chainId: 2522n, shortName: 'fraxtal-testnet' },
+    { chainId: 5330n, shortName: 'superseed' },
+    { chainId: 7897n, shortName: 'arena-z' },
+    { chainId: 695569n, shortName: 'pyrope' },
     { chainId: 195n, shortName: 'tokb' },
     { chainId: 196n, shortName: 'okb' },
     { chainId: 204n, shortName: 'opbnb' },
