diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index 3f070a3..b4b0768 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -1,6 +1,6 @@
-"use strict";
-Object.defineProperty(exports, "__esModule", { value: true });
-exports.networks = void 0;
+'use strict'
+Object.defineProperty(exports, '__esModule', { value: true })
+exports.networks = void 0
 // https://github.com/ethereum-lists/chains/tree/master/_data/chains
 exports.networks = [
     { chainId: 1n, shortName: 'eth' },
@@ -58,7 +58,7 @@ exports.networks = [
     { chainId: 336n, shortName: 'sdn' },
     { chainId: 338n, shortName: 'tcro' },
     { chainId: 420n, shortName: 'ogor' },
-    { chainId: 424n, shortName: 'PNG' },
+    { chainId: 424n, shortName: 'PGN' },
     { chainId: 570n, shortName: 'sys-rollux' },
     { chainId: 588n, shortName: 'metis-stardust' },
     { chainId: 592n, shortName: 'astr' },
@@ -66,6 +66,7 @@ exports.networks = [
     { chainId: 599n, shortName: 'metis-goerli' },
     { chainId: 686n, shortName: 'kar' },
     { chainId: 787n, shortName: 'aca' },
+    { chainId: 919n, shortName: 'modeTestnet' },
     { chainId: 1001n, shortName: 'baobab' },
     { chainId: 1008n, shortName: 'eun' },
     { chainId: 1030n, shortName: 'cfx' },
@@ -99,6 +100,7 @@ exports.networks = [
     { chainId: 2358n, shortName: 'kroma-sepolia' },
     { chainId: 3737n, shortName: 'csb' },
     { chainId: 4002n, shortName: 'tftm' },
+    { chainId: 4202n, shortName: 'lisksep' },
     { chainId: 4337n, shortName: 'beam' },
     { chainId: 4460n, shortName: 'orderlyl2' },
     { chainId: 4689n, shortName: 'iotex-mainnet' },
@@ -166,10 +168,12 @@ exports.networks = [
     { chainId: 622277n, shortName: 'rth' },
     { chainId: 7777777n, shortName: 'zora' },
     { chainId: 11155111n, shortName: 'sep' },
+    { chainId: 11155420n, shortName: 'opsep' },
     { chainId: 222000222n, shortName: 'kanazawa' },
     { chainId: 245022926n, shortName: 'neonevm-devnet' },
     { chainId: 245022934n, shortName: 'neonevm-mainnet' },
     { chainId: 333000333n, shortName: 'meld' },
+    { chainId: 999999999n, shortName: 'zsep' },
     { chainId: 1313161554n, shortName: 'aurora' },
     { chainId: 1313161555n, shortName: 'aurora-testnet' },
     { chainId: 1666600000n, shortName: 'hmy-s0' },
