diff --git a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
index 7dbddf5..545112f 100644
--- a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
+++ b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
@@ -45,12 +45,14 @@ exports.networks = [
     { chainId: 336, shortName: 'sdn' },
     { chainId: 338, shortName: 'tcro' },
     { chainId: 420, shortName: 'ogor' },
+    { chainId: 424, shortName: 'PGN' },
     { chainId: 588, shortName: 'metis-stardust' },
     { chainId: 592, shortName: 'astr' },
     { chainId: 595, shortName: 'maca' },
     { chainId: 599, shortName: 'metis-goerli' },
     { chainId: 686, shortName: 'kar' },
     { chainId: 787, shortName: 'aca' },
+    { chainId: 919, shortName: 'modeTestnet' },
     { chainId: 1001, shortName: 'baobab' },
     { chainId: 1008, shortName: 'eun' },
     { chainId: 1088, shortName: 'metis-andromeda' },
@@ -94,6 +96,7 @@ exports.networks = [
     { chainId: 12357, shortName: 'rei-testnet' },
     { chainId: 23294, shortName: 'sapphire' },
     { chainId: 23295, shortName: 'sapphire-testnet' },
+    { chainId: 34443, shortName: 'mode' },
     { chainId: 42161, shortName: 'arb1' },
     { chainId: 42170, shortName: 'arb-nova' },
     { chainId: 42220, shortName: 'celo' },
@@ -117,8 +120,11 @@ exports.networks = [
     { chainId: 421611, shortName: 'arb-rinkeby' },
     { chainId: 421613, shortName: 'arb-goerli' },
     { chainId: 534353, shortName: 'scr-alpha' },
+    { chainId: 7777777, shortName: 'zora' },
     { chainId: 11155111, shortName: 'sep' },
+    { chainId: 11155420, shortName: 'opsep' },
     { chainId: 245022926, shortName: 'neonevm-devnet' },
+    { chainId: 999999999, shortName: 'zsep' },
     { chainId: 1313161554, shortName: 'aurora' },
     { chainId: 1313161555, shortName: 'aurora-testnet' },
     { chainId: 1666600000, shortName: 'hmy-s0' },
