diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index c22dc1e..1ab785d 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -46,6 +46,7 @@ exports.networks = [
     { chainId: 148n, shortName: 'shimmerevm-mainnet' },
     { chainId: 155n, shortName: 'tenet-testnet' },
     { chainId: 169n, shortName: 'manta' },
+    { chainId: 185n, shortName: 'mint' },
     { chainId: 195n, shortName: 'tokb' },
     { chainId: 196n, shortName: 'okb' },
     { chainId: 204n, shortName: 'opbnb' },
@@ -64,7 +65,7 @@ exports.networks = [
     { chainId: 338n, shortName: 'tcro' },
     { chainId: 369n, shortName: 'pls' },
     { chainId: 420n, shortName: 'ogor' },
-    { chainId: 424n, shortName: 'PNG' },
+    { chainId: 424n, shortName: 'PGN' },
     { chainId: 570n, shortName: 'sys-rollux' },
     { chainId: 588n, shortName: 'metis-stardust' },
     { chainId: 592n, shortName: 'astr' },
@@ -111,6 +112,7 @@ exports.networks = [
     { chainId: 2222n, shortName: 'kava' },
     { chainId: 2331n, shortName: 'rss3-testnet' },
     { chainId: 2358n, shortName: 'kroma-sepolia' },
+    { chainId: 2523n, shortName: 'fraxtal-sep' },
     { chainId: 2810n, shortName: 'hmorph' },
     { chainId: 3636n, shortName: 'BTNX' },
     { chainId: 3737n, shortName: 'csb' },
@@ -136,6 +138,7 @@ exports.networks = [
     { chainId: 7171n, shortName: 'bitrock' },
     { chainId: 7332n, shortName: 'EON' },
     { chainId: 7341n, shortName: 'shyft' },
+    { chainId: 7560n, shortName: 'cyeth' },
     { chainId: 7700n, shortName: 'canto' },
     { chainId: 7771n, shortName: 'tbitrock' },
     { chainId: 8192n, shortName: 'tqf' },
@@ -216,6 +219,7 @@ exports.networks = [
     { chainId: 6038361n, shortName: 'azkyt' },
     { chainId: 11155111n, shortName: 'sep' },
     { chainId: 11155420n, shortName: 'opsep' },
+    { chainId: 111557560n, shortName: 'cysep' },
     { chainId: 94204209n, shortName: 'polygon-blackberry' },
     { chainId: 123420111n, shortName: 'opcelestia-raspberry' },
     { chainId: 168587773n, shortName: 'blastsepolia' },
