diff --git a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
index cfc40d8..71d0a34 100644
--- a/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
+++ b/node_modules/@safe-global/protocol-kit/dist/src/utils/eip-3770/config.js
@@ -48,6 +48,7 @@ exports.networks = [
     { chainId: 204n, shortName: 'opbnb' },
     { chainId: 246n, shortName: 'ewt' },
     { chainId: 250n, shortName: 'ftm' },
+    { chainId: 252n, shortName: 'fraxtal' },
     { chainId: 255n, shortName: 'kroma' },
     { chainId: 280n, shortName: 'zksync-goerli' },
     { chainId: 288n, shortName: 'boba' },
@@ -59,13 +60,14 @@ exports.networks = [
     { chainId: 336n, shortName: 'sdn' },
     { chainId: 338n, shortName: 'tcro' },
     { chainId: 420n, shortName: 'ogor' },
-    { chainId: 424n, shortName: 'PNG' },
+    { chainId: 424n, shortName: 'PGN' },
     { chainId: 570n, shortName: 'sys-rollux' },
     { chainId: 588n, shortName: 'metis-stardust' },
     { chainId: 592n, shortName: 'astr' },
     { chainId: 595n, shortName: 'maca' },
     { chainId: 599n, shortName: 'metis-goerli' },
     { chainId: 686n, shortName: 'kar' },
+    { chainId: 690n, shortName: 'redstone' },
     { chainId: 787n, shortName: 'aca' },
     { chainId: 919n, shortName: 'modesep' },
     { chainId: 1001n, shortName: 'baobab' },
@@ -102,6 +104,7 @@ exports.networks = [
     { chainId: 2358n, shortName: 'kroma-sepolia' },
     { chainId: 3737n, shortName: 'csb' },
     { chainId: 4002n, shortName: 'tftm' },
+    { chainId: 4202n, shortName: 'lisksep' },
     { chainId: 4337n, shortName: 'beam' },
     { chainId: 4460n, shortName: 'orderlyl2' },
     { chainId: 4689n, shortName: 'iotex-mainnet' },
@@ -134,6 +137,7 @@ exports.networks = [
     { chainId: 12357n, shortName: 'rei-testnet' },
     { chainId: 13337n, shortName: 'beam-testnet' },
     { chainId: 17000n, shortName: 'holesky' },
+    { chainId: 17069n, shortName: 'garnet' },
     { chainId: 17172n, shortName: 'eclipse' },
     { chainId: 18231n, shortName: 'unreal' },
     { chainId: 23294n, shortName: 'sapphire' },
